elastic/kibana
# [Telemetry] Replace usage of old settings field

Addresses https://github.com/elastic/kibana/issues/172922

## Summary

The `src/plugins/advanced_settings` plugin will soon be removed and completely replaced by the new settings packages: `packages/kbn-management/settings`. This PR replaces the usages of the old Advanced settings `Field` component in the telemetry plugin. 

<img width="1217" alt="Screenshot 2024-01-12 at 15 02 56" src="https://github.com/elastic/kibana/assets/59341489/45a4a084-33ad-425c-b137-5acfe3c8bd3d">

**How to test:**
1. Start Es and Kibana
2. Go to Stack Management -> Advanced settings -> Global Settings
3. Verify that the "Usage collection" setting field renders and works as expected.

Note: There is also a [usage](https://github.com/elastic/kibana/blob/5a32eb0a7f56e53b1ade3f85311694ddc203dd28/src/plugins/telemetry_management_section/public/plugin.tsx#L20) of `AdvancedSettingsSetup` in the telemetry setup contract - this will be replaced in a future PR when we register the Advanced settings application from the `management` plugin.
